common/src/main/java/io/netty/util/collection/PrimitiveCollections.java

```diff
@@ -50,6 +52,23 @@ private PrimitiveCollections() {
      * An empty map. All operations that attempt to modify the map are unsupported.
      */
     private static final class EmptyIntObjectMap implements IntObjectMap<Object> {
```

Comment:
Should `EmptyIntObjectMap` be "script templated" like the other classes?  Also the associated accessors in this class?